addons/web/static/src/views/graph/graph_renderer.xml

```diff
@@ -24,7 +24,7 @@
 
     <t t-name="web.GraphRenderer" owl="1">
         <div class="o_graph_renderer">
-            <label t-if="props.metaData.title" t-esc="props.metaData.title"/>
+            <!-- <label t-if="props.metaData.title" t-esc="props.metaData.title"/> -->
```

Comment:
problem with style --> we should review the use of string/title attribute on arch through time (from 12.0 to now)